lib/Factory/ModuleTemplateFactory.php

```diff
@@ -168,7 +168,7 @@ private function createFromXml(\DOMElement $xml): ModuleTemplate
         $template->dataType = $this->getFirstValueOrDefaultFromXmlNode($xml, 'dataType');
         $template->title = $this->getFirstValueOrDefaultFromXmlNode($xml, 'title');
         $template->thumbnail = $this->getFirstValueOrDefaultFromXmlNode($xml, 'thumbnail');
-        $template->isVisible = $this->getFirstValueOrDefaultFromXmlNode($xml, 'isVisible');
+        $template->isVisible = $this->getFirstValueOrDefaultFromXmlNode($xml, 'isVisible') === 'true';
```

Comment:
This means that if it is empty (not provided in the XML) it will be visible false - I think logically we'd want this the other way around, i.e. if it isn't there then it should be visible=true?